web/src/app/administration/access-items-management/access-items-management.component.ts

```diff
@@ -155,6 +155,14 @@ export class AccessItemsManagementComponent implements OnInit, OnDestroy {
     );
   }
 
+  private addIfExists(item?: string) {
```

Comment:
is there a better way? don't like meaningless catches

Reply:
thanks for asking - I forgot to remove this changes!